starknet_py/tests/e2e/client_devnet/time_test.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_time_advancing(devnet_client):
+    time = 3384617820
+    step, error = 100, 3
+
+    await devnet_client.set_time(time, generate_block=True)
+    block = await devnet_client.get_block(block_number="latest")
+
+    assert block.timestamp == time
+
+    await devnet_client.increase_time(step)
+    block = await devnet_client.get_block(block_number="latest")
+
+    assert block.timestamp <= time + step + error
+    assert block.timestamp >= time + step
+
+
+@pytest.mark.asyncio
+async def test_time_set_with_generate_block(devnet_client):
```

Comment:
nit:
```suggestion
async def test_set_time_with_generate_block(devnet_client):
```
The same suggestion in the test below